packages/grafana-eslint-rules/README.md

```diff
@@ -111,3 +111,11 @@ const getStyles = (theme: GrafanaTheme2) => ({
 ### `theme-token-usage`
 
 Used to find all instances of `theme` tokens being used in the codebase and emit the counts as metrics. Should **not** be used as an actual lint rule!
+
+### `no-untranslated-literals`
+
+Check if strings are marked for translation.
+
+Right now, we only check if a string is wrapped up by the `Trans` tag. We currently do not apply this rule to props, attributes or similars.
```

Comment:
Check out the good docs from @ashharrison90 for the `no-unreduced-motion` rule - as these docs are linked to from the lint error message, they should be written with the goal of helping the developer fix their code.

I think they should (very briefly) explain how to use `<Trans />` or `t()` to translate a phrase, and then link to the rest of the i18n docs like you have.

Reply:
i think we could shift this to the bottom. as a consumer of the rule i don't care about the implementation details so much 🤔